CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Enable preferRest option by default for Firestore functions. (#6147)
```

Comment:
```suggestion
- Enable [preferRest](https://firebase.google.com/docs/reference/admin/node/firebase-admin.firestore.firestoresettings.md#firestoresettingspreferrest) option by default for Firestore functions. (#6147)
```

Reply:
This sounds great!  We need to remember to get this listed in the release notes, with the added explanation that the optimizations will always apply except when gRPC is used.